docs/cli-reference/ks_show.md

```diff
@@ -1,14 +1,37 @@
 ## ks show
 
-Show expanded resource definitions
+Show expanded manifests for a specific environment.
 
 ### Synopsis
 
 
-Show expanded resource definitions
+Show expanded manifests (resource definitions) for a specific environment. Jsonnet manifests,
+each defining a ksonnet component, are expanded into their JSON or YAML equivalents (YAML is the default).
+Any parameters for a component are resolved based on environment-specific values.
```

Comment:
I think this sentence would flow better, if placed after the next paragraph's `...directory into a list of resource definitions.`

Reply:
Hm..I see what you're saying, but the next paragraph is specific to when no component is specified. (Whereas this sentence discusses parameters, which are more general---i.e. they are resolved whether you use `-c`command or not).

I'll try rephrasing this for flow though.